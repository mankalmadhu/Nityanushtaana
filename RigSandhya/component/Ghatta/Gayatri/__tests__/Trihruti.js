import React from 'react';
import ತ್ರಿಹೃತಿ from '../Trihruti';
import Renderer from 'react-test-renderer';



jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');

test("ತ್ರಿಹೃತಿ", () => {
   expect(Renderer.create(
    <ತ್ರಿಹೃತಿ />
  ).toJSON()).toMatchSnapshot();

});