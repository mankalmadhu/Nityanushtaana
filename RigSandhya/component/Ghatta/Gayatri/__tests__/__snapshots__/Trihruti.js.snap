// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ತ್ರಿಹೃತಿ 1`] = `
<ವಿನಿಯೋಗ>
  ಓ೦ ಭೂರ್ಭುವ॒ಸ್ಸ್ವಃ । 
</ವಿನಿಯೋಗ>
`;
